src/style-spec/reference/v8.json

```diff
@@ -4450,25 +4450,19 @@
       },
       "property-type": "data-driven"
     },
-    "circle-ang-start": {
+    "circle-start-angle": {
       "type": "number",
       "default": 0,
       "minimum": 0,
       "transition": true,
-      "units": "angle",
-      "doc": "Angle value to start drawing a circle.",
+      "units": "degrees",
+      "doc": "Angle value to start drawing a circle. The circle is represented as a coordinate system relative to the viewport and 0° - value of the beginning of the first quadrant. Then drawing goes counter clockwise.",
```

Comment:
👍

Reply:
I'm a bit confused by "relative to the viewport and 0°". For other rotation values, we have a rotation alignment property to specify either relative to the viewport or relative to the map eg. https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/#layout-symbol-icon-rotation-alignment. So I think for conistancy we should either also have an angle-alignment property which is map or viewport, or specify in the description if this is map or viewport.

Although I think most of the time you'd want a map alignment (eg. direction range of a lookout/viewpoint), you may also sometimes want a viewport alignment?

Reply:
That makes sense, based on the [initial document](https://iho.int/uploads/user/pubs/standards/s-4/S4_V4-8-0_Oct_2018_EN.pdf) it looks like the initial intention was to support it relative to the map, but having options would be more consistent. @ws-hofshteyn have you tested this on tilted views?

Reply:
> have you tested this on tilted views?

and rotated views